next-frontend/src/lib/wca/wcaAPI.ts

```diff
@@ -0,0 +1,5 @@
+import createClient from "openapi-fetch";
+import type { paths } from "./wcaSchema"; // generated by openapi-typescript
+
+export const unauthenticatedClient = createClient<paths>({ baseUrl: "http://localhost:3000/api/v0/" });
```

Comment:
Why does the authorized one get `application/json` headers, but the unauthenticated version doesn't?

Reply:
good point